### PR TITLE
feat: add block number to quote report data [TKR-314]

### PR DESCRIPTION
**Pull request by kimpers**
# Description
Asset swapper changes 0xProject/protocol#448

**TODO** 
- [x] update with new  version of asset-swapper

# Testing Instructions

# Checklist

-   [ ] Update [documentation](https://github.com/0xProject/website/blob/development/mdx/api/index.mdx) as needed. **Website Documentation PR:**
-   [ ] Prefix PR title with `[WIP]` if necessary.
-   [ ] Add tests to cover changes as needed.
-   [ ] Test changes on the staging environment with [Matcha API Staging](https://api-staging.matcha.xyz).

    -   [ ] SRA/Limit orders
    -   [ ] Swap endpoints
    -   [ ] Meta transaction endpoints
    -   [ ] Depth charts

    For more information see `0x API Matcha smoke test runbook` in Quip.

### CODE REVIEW

File: src/utils/quote_report_utils.ts [hunk 1]
```diff
@@ -132,15 +134,20 @@ export const quoteReportUtils = {
...
+                .send({
+                    topic: KAFKA_TOPIC_QUOTE_REPORT,
+                    messages: [
+                        {
+                            value: JSON.stringify(extendedQuoteReport),
+                        },
+                    ],
+                })
+                .catch((err) => {
+                    logger.error(`Error publishing quote report to Kafka: ${err}`);
```
Comment (kimpers) [inline src/utils/quote_report_utils.ts:149]: I suppose we would like to know when this fails

**Review by dorothy-zbornak — APPROVED** [review]

**Comment by kimpers** [comment]
deploy staging

**Comment by kimpers** [comment]
deploy staging

**Comment by kimpers** [comment]
deploy staging

**Comment by kimpers** [comment]
deploy staging

**Comment by kimpers** [comment]
deploy polygon production

**Comment by kimpers** [comment]
deploy production

**Comment by kimpers** [comment]
deploy bsc production

**Comment by kimpers** [comment]
deploy polygon production

**Comment by kimpers** [comment]
deploy avalanche production

**Comment by kimpers** [comment]
deploy celo production

**Pull request merged**